libraries/SPI/src/SPI.cpp

```diff
@@ -421,3 +421,41 @@ void SPIClass::detachInterrupt(void)
 {
   // Should be disableInterrupt()
 }
+
+#if defined(SUBGHZSPI_BASE)
+SUBGHZSPIClass SubGHZ_SPI;
+
+void SUBGHZSPIClass::begin(uint8_t _pin)
+{
+  if (_pin != CS_PIN_CONTROLLED_BY_USER) {
+    LL_PWR_SelectSUBGHZSPI_NSS();
+  }
+  SPIClass::begin(_pin);
+  if (_pin != CS_PIN_CONTROLLED_BY_USER) {
+    LL_PWR_UnselectSUBGHZSPI_NSS();
+  }
+}
+
+byte SUBGHZSPIClass::transfer(uint8_t _pin, uint8_t _data, SPITransferMode _mode)
```

Comment:
This only overrides a single `transfer()` overload, but there are three more (`transfer16` plus two versions that take buffer pointers) and a `beginTransaction()` method that all control the passed pin.

I wonder if, instead of overriding these methods, it would make sense to introduce a single (virtual) `controlNss()` method or so, that is called in all these places (maybe also from `begin()` for the initial `pinMode`) and can be overridden by `SUBGHZSPIClass`?

Reply:
Should be ok now.